src/datumaro/plugins/data_formats/kaggle/base.py

```diff
@@ -125,7 +125,11 @@ def _load_items(self, ann_file: str, columns: Dict[str, Union[str, list]]):
 
         indices = {"media": df_fields.index(columns["media"])}
         if "label" in columns:
-            indices.update({"label": df_fields.index(columns["label"])})
+            labels = columns["label"]
+            if not isinstance(labels, list):
+                labels = [labels]
+            for label in labels:
+                indices.update({"label": df_fields.index(label)})
```

Comment:
indices["label"] is just pointing the index of the last label. Is it intended?

Reply:
I think we need a test code, too.

Reply:
Fixed in [e07489b](https://github.com/openvinotoolkit/datumaro/pull/1607/commits/e07489b4eb8bd2fc212070c39b4062f882793d66) and added unit tests in [3619792](https://github.com/openvinotoolkit/datumaro/pull/1607/commits/3619792653d51caebc92b4fbeee95789fbdb6dfa)